camviewer_ui_impl.py

```diff
@@ -2026,6 +2038,21 @@ def connectCamera(self, sCameraPv, index, sNotifyPv=None):
         # Get camera configuration
         self.getConfig()
 
+    def setup_model_specific(self):
+        if self.cam_type_screen_generator is None:
+            form = QFormLayout()
+            self.ui.groupBoxControls.setLayout(form)
+        else:
+            self.cam_type_screen_generator.cleanup()
+            form = self.ui.groupBoxControls.layout()
+        self.cam_type_screen_generator = CamTypeScreenGenerator(self.ctrlBase, form)
```

Comment:
So every time we set a new camera, we create a new `CamTypeScreenGenerator`?  Do we ever unparent the old ones or deleteLater them? 

In flipping between cameras I never ran into a problem/memory leak, but I am suspicious.  It seems like we're already disconnecting signals and removing rows from the form, so could we just repopulate it?

Ultimately I think I'm ok with this as is.  If we run into performance issues we can revisit.

Reply:
You are correct, replacing this object should no longer be necessary with some small tweaks. (I won't fix this now, in favor of forward progress through the last few camviewer tickets)

In an earlier version of this PR I was trying to remove the `QFormLayout` from the `QGroupBox` which actually seems to be impossible in pyqt. In that earlier version the screen generator was a one-time use that had custody of a unique layout per screen generator object, so I was removing and replacing them. It was only later that I switched to clearing the layout instead.